src/Parser/HolyPriest/Modules/Spells/DivineHymn.js

```diff
@@ -21,6 +21,28 @@ class DivineHymn extends Module {
     }
   }
 
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.DIVINE_HYMN_CAST.id) {
+      return;
+    }
+
+    this.casts += 1;
+  }
+
+  suggestions(when) {
+    const missedHymnTicks = (this.casts * 5) - this.ticks;
+
+    when(missedHymnTicks).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('You wasted Divine Hymn ticks. Try to avoid clipping the end of Divine Hymn as well as positioning such that you will not have to move during its duration. ')
+          .icon('spell_holy_divinehymn')
+          .actual(`${actual} missed Hymn ticks`)
+          .recommended(`0 is recommended`)
+          .regular(recommended).major(recommended);
```

Comment:
I think different thresholds could still make sense for this statistic, especially if you changed the trigger to missedHymnTicksPerMinute or missedHymnTicksPerCast so that it works about the same on a 3 minute fight as it would on a 15 minute fight. This way all suggestions are always marked as major issues, and you generally want to avoid having the suggestions nag when it's actually not that important to put more emphasis on those suggestions that do really matter. Even more so since this ability is especially hard to get a full cast out on some fights, allowing for some slack shouldn't make that big a difference.